packages/webdriver/src/command.js

```diff
@@ -88,4 +88,12 @@ export default function (method, endpointUri, commandInfo) {
             return result.value
         })
     }
+
+    /**
+     * skip certain protocol commands as far as these commands are wrapped by WebdriverIO
```

Comment:
can you elaborate on this?

Reply:
I hove the table might help
```
Protocol Command			WebdriverIO Command
-----------------------------------------------
findElement					$
findElements				$$
findElementFromElement		$
findElementsFromElement 	$$
getActiveElement			?
isElementSelected			isSelected
isElementDisplayed			isDisplayed
getElementAttribute			getAttribute
getElementProperty			getProperty
getElementCSSValue			getCSSProperty
getElementText				getText
getElementTagName			getTagName
getElementLocation			getLocation
getElementRect				getSize
isElementEnabled			isEnabled
elementClick				click
elementClear				clearValue
elementSendKeys				addValue
takeElementScreenshot		saveScreenshot

getElementLocationInView	?
getActiveElement			?
elementEquals				?
elementSubmit				?
```

Reply:
I might understood the question wrong, if you like we may discuss it offline or you may be more specific.